tokio/src/runtime/blocking/pool.rs

```diff
@@ -126,10 +131,15 @@ impl BlockingPool {
         shared.shutdown = true;
         shared.shutdown_tx = None;
         self.spawner.inner.condvar.notify_all();
+        let workers = shared.worker_threads.drain().collect::<Vec<_>>();
```

Comment:
This could be something like:

```rust
let workers = mem::replace(&mut shared.worker_threads, Slab::new());
```

This would avoid having to allocate a new Vec for storage.

Reply:
Nice little trick. Done.